docs/docs/Components/components-helpers.md

```diff
@@ -114,25 +129,30 @@ In this example, the **Message Store** component stores the complete chat histor
 
 For more information on configuring memory in Langflow, see [Memory](/memory).
 
-### Inputs
+<details>
+<summary>Parameters</summary>
+
+**Inputs**
+
+| Name | Type | Description |
+|------|------|-------------|
+| memory | Memory | Retrieve messages from an external memory. If empty, it will use the Langflow tables. |
+| sender | String | Filter by sender type. |
+| sender_name | String | Filter by sender name. |
+| n_messages | Integer | The number of messages to retrieve. |
+| session_id | String | The session ID of the chat. If empty, the current session ID parameter will be used. |
```

Comment:
Avoid future tense unless you need to specify that an action will take place in the future. Apply throughout the PR.